include/ck_tile/ops/gemm/pipeline/block_gemm_pipeline_problem.hpp

```diff
@@ -33,4 +40,43 @@ struct BlockGemmPipelineProblem
     static constexpr index_t AlignmentC = kPadC ? 1 : VectorLoadSize / sizeof(CDataType);
 };
 
+template <typename ADataType_,
+          typename BDataType_,
+          typename CDataType_,
+          typename BlockGemmShape_,
+          typename ALayout_,
+          typename BLayout_,
+          typename CLayout_,
+          bool kPadA_                      = false,
+          bool kPadB_                      = false,
+          bool kPadC_                      = false,
+          GemmPipelineScheduler Scheduler_ = GemmPipelineScheduler::Intrawave,
+          bool HasHotLoop_                 = true,
+          TailNumber TailNum_              = TailNumber::Full>
+struct UniversalGemmPipelineProblem
```

Comment:
Do we need separate Problem structure for universal gemm? Maybe we can update previous version with some default parameters

Reply:
Hi Adam, Cheers for this new PR! Please allow me to take some time to review this PR. I will finish the review by this weekend or next Monday. FYI on this issue. I discussed with Qianfeng and Yao Dan today. I will modify the fmha to let it calling from the Block level problem of the GEMM instead of the Pipeline Problem. I also rename the Pipeline name to delete all the confusing "Block" name in the pipeline file.